vaporetto/src/sentence.rs

```diff
@@ -1203,6 +1209,31 @@ impl<'a, 'b> Token<'a, 'b> {
         &self.sentence.tags[start..end]
     }
 
+    /// Returns tag candidates with scores.
+    ///
+    /// The return value is a two-dimensional array. The outer array index corresponding to the
+    /// return value of [`Token::tags()`]. The inner array is a candidate set, where each element
+    /// is a tuple of the tag name and its score.
+    pub fn tag_candidates(&self) -> Vec<Vec<(&'b str, i32)>> {
```

Comment:
```suggestion
    #[cfg(feature = "tag-prediction")]
    pub fn tag_candidates(&self) -> Vec<Vec<(&'b str, i32)>> {
```
I guess that this function is never used unless`tag-prediction` is specified. The same is true for `Token::tags`.

Reply:
I see